### ISSUE TITLE
the performance on SUN397dataset is strange

### ISSUE THREAD

**Issue by John-niu-07** [apanariello4/core-space-merging#3]
Dear Authors,
  I run your code and found the acc on SUN397 is about 20% (abs) and 32% (normal). In contrast, the paper report the normal acc is about 99% on SUNs97 dataset (refer to Tab 3). Could you help me on it?
 Thank you so much

**Comment by apanariello4** [comment]
Dear John-niu-07,

check for the following:
- did you download the dataset correctly? you can verify by checking the abs accuracy of the finetuned model. If not, dataset or checkpoint loading may be wrong
- what kind of method are you running?

**Comment by John-niu-07** [comment]
Thanks, 
 1) I think I have download the right dataset (with the sun397.py in the dataset folds), the abs acc is 20% while the normal acc is 32%, while the fine-tuned acc is 64.5%. I run both Full+TSV and Core+TSV, and both of them have very low acc :(

Thanks for your responses

**Comment by apanariello4** [comment]
Can you share the actual command you are running to get these results?
